dist/general.go

```diff
@@ -6,20 +6,17 @@ type Parameter struct {
 	Value float64
 }
 
-// A ParameterMarshaler is a type that can marshal itself into a slice of Paramaters
-// and unmarshal itself from the slice of parameters.
-// ParameterMarshaler exists to support algorithms that modify parameters of arbitrary distributions.
-// Typically, users should modify distributions using the fields of the specifc
-// distribution.
-// Marshal and Unmarshal are to be used as a pair, users should not attempt to construct
+// A parameterMutator is a type that can both get and set the parameters of a distribution.
+// Typically, users should modify distributions using the fields of the specifc distribution.
```

Comment:
I don't think this "users" comment is necessary now that it's an internal function. There aren't any users involved.